core-bundle/src/Resources/contao/templates/frontend/fe_page.html5

```diff
@@ -9,7 +9,7 @@
 
     <?php $this->block('meta'); ?>
       <meta name="robots" content="<?= $this->robots ?>">
-      <meta name="description" content="<?= $this->description ?>">
+      <meta name="description" content="<?= mb_substr($this->description, 0, 320) ?>">
```

Comment:
That would be a behaviour change. You need to use `StringUtil::substr($strText, 320)`, otherwise the ellipsis won't get added.

Reply:
Yes, which we don't want. It makes no sense at all and `StringUtil::substr()` does way too much bad stuff. E.g. it's also calling `strip_tags()` for who knows what reason.

Reply:
> It makes no sense at all

Imho it does make sense to create a description that reads

> Auch gibt es niemanden, der den Schmerz an sich liebt, sucht oder wünscht, nur, weil er Schmerz ist, es sei denn, es kommt zu zufälligen Umständen, in denen Mühen und Schmerz ihm große Freude bereiten können. Um ein triviales Beispiel zu nehmen, wer von uns unterzieht sich je anstrengender körperlicher Betätigung, außer …

instead of

> Auch gibt es niemanden, der den Schmerz an sich liebt, sucht oder wünscht, nur, weil er Schmerz ist, es sei denn, es kommt zu zufälligen Umständen, in denen Mühen und Schmerz ihm große Freude bereiten können. Um ein triviales Beispiel zu nehmen, wer von uns unterzieht sich je anstrengender körperlicher Betätigung, auße

Reply:
> E.g. it's also calling strip_tags() for who knows what reason.

It strips tags to avoid creating invalid HTML. If you want to clip text with HTML, you need to use `StringUtil::substrHtml`.